### PR TITLE
RavenDB-22415 

### PR DESCRIPTION
**Pull request by gregolsky**
### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-22415

### Additional description

- move the default value for UnsecuredAccessAllowed from settings to env var
- v5.4 should not use `latest` tag

### Type of change

- [x] Bug fix
- [ ] Regression bug fix
- [x] Optimization
- [ ] New feature

### How risky is the change?

- [ ] Low 
- [x] Moderate 
- [ ] High
- [ ] Not relevant

### Backward compatibility

- [ ] Non breaking change
- [x] Ensured. 

Instead of using a value in settings.json, which is hard and not intuitive to override we use a default value declared as ENV in the dockerfile, which is something that is less confusing and more discoverable.

- [ ] Breaking change
- [ ] Not relevant

### Is it platform specific issue?

- [x] Yes. COntainers
- [ ] No

### Documentation update

- [x] This change requires a documentation update. Please mark the issue on YouTrack using `Documentation Required` tag.
- [ ] No documentation update is needed 

We should check if we have proper docs explaining how to run unsecured in docker if someone is just running a POC.

### Testing by Contributor

- [ ] Tests have been added that prove the fix is effective or that the feature works
- [ ] Internal classes added to the test class (e.g. entity or index definition classes) have the lowest possible access modifier (preferable `private`) 
- [x] It has been verified by manual testing

I built it on my local and tested the Ubuntu image (with and without the Unsecured... option set):
```
$ docker run -it -p 8080:8080 -e RAVEN_ServerUrl=http://0.0.0.0:8080 -e RAVEN_Security_UnsecuredAccessAllowed=PublicNetwork -e RAVEN_Setup_Mode=None ravendb/ravendb:latest-lts
```

I was not successful in testing the Windows image. Something seems to be broken on my laptop. @poissoncorp @hiddenshadow21 Can you check on your environment please? 

### Testing by RavenDB QA team

- [ ] This change requires a special QA testing due to possible performance or resources usage implications (CPU, memory, IO). Please mark the issue on YouTrack using `QA Required` tag.
- [x] No special testing by RavenDB QA team is needed

### Is there any existing behavior change of other features due to this change?

- [ ] Yes. Please list the affected features/subsystems and provide appropriate explanation
- [x] No

The flag works as it's been before - it is just defined elsewhere.

### UI work

- [ ] It requires further work in the Studio. Please mark the issue on YouTrack using `Studio Required` tag.
- [x] No UI work is needed

### CODE REVIEW

**Comment by gregolsky** [comment]
cc @karmeli87 this should make new Docker image users' life a little bit easier :)

File: docker/common.ps1 [hunk 1]
```diff
@@ -35,7 +35,6 @@ function GetUbuntuImageTags($repo, $version, $arch) {
     switch ($arch) {
         "x64" { 
             return @(
-                "$($repo):latest",
                 "$($repo):ubuntu-latest",
```
Comment (arekpalinski) [inline docker/common.ps1]: Is `ubuntu-latest` ok here?

**Review by hiddenshadow21 — APPROVED** [review]
Building windows image on my PC works fine.

**Review by ppekrol — APPROVED** [review]

**Comment by poissoncorp** [comment]
Works on my PC inside the Windows container too.

**Review by poissoncorp — APPROVED** [review]

**Pull request merged**